dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -2,5 +2,13 @@ class Pd::ProfessionalLearningLandingController < ApplicationController
   before_action :require_admin
 
   def index
+    # Get workshops that the user is enrolled in with an outstanding survey
+    workshops = Pd::Workshop.enrolled_in_by(current_user)
```

Comment:
This comment doesn't match the code. This will get all workshops the user is enrolled in regardless of survey.